### PR TITLE
add tezt.2.0.0

### PR DESCRIPTION
**Pull request by rbardou**
Second release of Tezt.

### CODE REVIEW

File: packages/tezt/tezt.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.9" }
+  "ocaml" { >= "4.08" }
```
Comment (mseri) [inline packages/tezt/tezt.2.0.0/opam:10]: Used Sys.rmdir which was added in 4.12

Comment (mseri) [reply packages/tezt/tezt.2.0.0/opam:10]: @rbardou Can I merge this change or you want to make it compatible with 4.08-4.11?

Comment (doomeer) [reply packages/tezt/tezt.2.0.0/opam:10]: Sorry for the late reply — yes please go ahead and merge this change, thanks for the analysis! I'll fix this upstream.

Comment (rbardou) [reply packages/tezt/tezt.2.0.0/opam:10]: Sorry for the late reply — yes please go ahead and merge this change, thanks for the analysis! I'll fix this upstream.

**Comment by mseri** [comment]
Thanks

**Pull request merged**